sys/net/network_layer/fib/fib.c

```diff
@@ -472,16 +482,29 @@ void fib_deinit(void)
     mutex_unlock(&mtx_access);
 }
 
-void fib_register_rrp(void)
+int fib_register_rrp(uint8_t *prefix, size_t prefix_size)
 {
     mutex_lock(&mtx_access);
 
+    if (notify_rrp_pos >= FIB_MAX_RRP) {
+        mutex_unlock(&mtx_access);
+        return -ENOMEM;
+    }
+
+    if ((prefix == NULL) || (prefix_size == 0)) {
```

Comment:
If a RRP wants to "catch all", it can provide an all `0` prefix. 
However, this is usually the default gateway and should be determined for instance on 6LoWPAN level. A routing protocol is usually responsible for a specific prefix and not the complete `::/128` range.
But technically (after #2783 has been merged), you can just tell the FIB this way to inform you always on unreachable destinations.